src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -261,8 +263,8 @@ function useChatUtilsHook(): UseChatUtilsHook {
     }));
   };
 
-  const goToContactScreen = () => {
-    navigation.navigate('ContactScreen');
+  const goToContactScreen = ({from}: ContactScreenPayload) => {
+    navigation.navigate('ContactScreen', {from});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `goToContactScreen` function now accepts a payload object with a `from` property. This change allows for more flexibility in navigation, as the source of the navigation can now be passed as a parameter. However, ensure that all calls to this function are updated to pass the required parameter.
```diff
- const goToContactScreen = () => {
-   navigation.navigate('ContactScreen');
+ const goToContactScreen = ({from}: ContactScreenPayload) => {
+   navigation.navigate('ContactScreen', {from});
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->